tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -570,7 +570,13 @@ public function createExternalAPI() {
    */
   protected function createTestEntity(string $entity, array $params, string $identifier = 'default') {
     $params['version'] = 4;
-    $result = $this->callAPISuccess($entity, 'create', $params);
+    $params['debug'] = TRUE;
```

Comment:
what if you set $this->_version = 4 before $this->callAPISuccess or does that break other things?

Reply:
I still don't think that's necessary. Explicitly passing version in to the function like this ought to work.